src/StatusQ/Controls/StatusInputWithCursor.qml

```diff
@@ -0,0 +1,62 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+StatusInput {
+    id: cursorInput
+
+    property string cursorColor: Theme.palette.primaryColor1
+
+    height: input.edit.height
+    leftPadding: 0
+    rightPadding: 0
+
+    input.placeholderText: ""
+    input.edit.cursorVisible: true
+    input.edit.font.pixelSize: 32
+    input.placeholderFont.pixelSize: 32
+    input.leftPadding: 0
+    input.rightPadding: 0
+    input.topPadding: 0
+    input.bottomPadding: 0
+    input.edit.padding: 0
+    input.background.color: "transparent"
+    input.background.border.width: 0
+    input.edit.cursorDelegate: Rectangle {
+        id: cursor
+        visible: input.edit.cursorVisible
+        color: cursorColor
+        width: 2
+
+        SequentialAnimation {
+            loops: Animation.Infinite
+            running: input.edit.cursorVisible
+
+            PropertyAction {
+                target: cursor
+                property: 'visible'
+                value: true
+            }
+
+            PauseAnimation {
+                duration: 600
+            }
+
+            PropertyAction {
+                target: cursor
+                property: 'visible'
+                value: false
+            }
+
+            PauseAnimation {
+                duration: 600
+            }
+
+            onStopped: {
+                cursor.visible = false
+            }
+        }
+    }
+}
```

Comment:
Does this component need documentation?

Reply:
Have we started documenting statusq components already? 😮

Reply:
We have! :D